.github/workflows/build.yml

```diff
@@ -0,0 +1,103 @@
+      run: |
+        wget -q https://downloads.tuxfamily.org/godotengine/${{ inputs.godot-version }}/Godot_v${{ inputs.godot-version }}-stable_linux_headless.64.zip
+        wget -q https://downloads.tuxfamily.org/godotengine/${{ inputs.godot-version }}/Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+    - name: Add dependencies to the cache if necessary
+      if: steps.cache-godot.outputs.cache-hit != 'true'
+      uses: actions/cache/save@v4
+      with:
+        path: |
+          Godot_v${{ inputs.godot-version }}-stable_linux_headless.64.zip
+          Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+        key: ${{ steps.cache-godot.outputs.cache-primary-key }}
+    - name: Install Godot and the export templates
+      run: |
+        unzip Godot_v${{ inputs.godot-version }}-stable_linux_headless.64.zip
+        rm Godot_v${{ inputs.godot-version }}-stable_linux_headless.64.zip
+        unzip Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+        rm Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+        mkdir -v -p ~/.local/share/godot/templates/${{ inputs.godot-version }}.stable/
+        mv templates/* ~/.local/share/godot/templates/${{ inputs.godot-version }}.stable/
+        ls ~/.local/share/godot/templates/${{ inputs.godot-version }}.stable/
+    - name: Install mingw
+      run: sudo apt-get install gcc-mingw-w64
+    - name: Create the output directories
+      run: |
+        mkdir output
+        mkdir output/burrito_link
+    - name: Build X11_FG
+      run: |
+        cd burrito-fg
+        cargo build --release
+    - name: Build taco_parser
+      run: |
+        cd taco_parser
+        cargo build --release
+    - name: Build Burrito Link
+      run: |
+        mkdir burrito_link/build
+        cd burrito_link/build
+        cmake ..
+        make
+    - name: Build Burrito
+      run: |
+        mkdir build
+        ./Godot_v${{ inputs.godot-version }}-stable_linux_headless.64 --export "Linux/X11"
+        chmod +x build/burrito.x86_64
+    - name: Fill the output folder
+      run: |
+        mv burrito_link/build/burrito_link.exe output/burrito_link
+        mv burrito_link/build/d3d11.dll output/burrito_link
+        mv build/burrito.x86_64 output/
+        mv build/libburrito_fg.so output/
+        mv build/libgw2_taco_parser.so output/
+    - name: Upload output folder as an artifact
+      uses: actions/upload-artifact@v4
+      id: artifact-upload
+      with:
+        name: burrito-binaries
+        path: output
+        if-no-files-found: error
+        overwrite: true
+    - name: Strip the burrito_link binaries for reproducibility
+      run: strip output/burrito_link/*
+    - name: Get the md5 sums for the produced output files
+      id: final
+      run: |
+        mv output/burrito_link/* output/
+        rm -rf output/burrito_link
+        echo 'checksum<<EOF' >> $GITHUB_OUTPUT
+        md5sum output/* >> $GITHUB_OUTPUT
```

Comment:
Can this be `sha256sum` instead, along with a change to the other 2 mentions of MD5, or maybe even both?